src/components/UI/DeleteButton/DeleteButton.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Button } from "carbon-components-react";
+
+const DeleteButton = (props) => {
+  return (
+    <div className="delete-button__button">
+      <Button onClick={props.deleteCardsHandler}>Delete Cards</Button>
```

Comment:
For deletion - it's not like the planned primary action on the page. Better use secondary type.